### PR TITLE
Irinata/clean

### PR DESCRIPTION
**Pull request by irinatarn**
Restricting user history transactions to 10 or to the last week since this will shrink the payload significantly.

### CODE REVIEW

File: backend/queues/src/main/resources/application.yml [hunk 1]
```diff
@@ -121,6 +121,7 @@ azure:
     app-service-principal-url: https://graph.microsoft.com/v1.0/servicePrincipals/${DFP_SP_ID}
     user-photo-url-template: https://graph.microsoft.com/beta/users/#user_id#/photo/$value
     timeout: PT5S
+    previous-transactions-count: 10
```
Comment (satyamamit) [inline backend/queues/src/main/resources/application.yml:124]: as discussed, this number will not work well for all calculations.

**Review by satyamamit — CHANGES_REQUESTED** [review]

File: backend/queues/src/main/resources/application.yml (hunk 1 above)
Comment (satyamamit) [inline backend/queues/src/main/resources/application.yml:124]: why did you keep 10 here ?

Comment (irinatarn) [reply backend/queues/src/main/resources/application.yml:124]: this is local:)

File: backend/queues/src/main/java/com/griddynamics/msd365fp/manualreview/queues/service/ItemEnrichmentService.java [hunk 2]
```diff
@@ -409,7 +412,9 @@ private void mapPIEntitiesToPurchase(final Purchase purchase, final List<Explore
         if (purchase.getAddressList() == null) {
             purchase.setAddressList(new LinkedList<>());
         }
-        purchase.getAddressList().addAll(billingAddressMap.values());
```
Comment (satyamamit) [inline backend/queues/src/main/java/com/griddynamics/msd365fp/manualreview/queues/service/ItemEnrichmentService.java:412]: what is difference between add and addAll ?

Comment (irinatarn) [reply backend/queues/src/main/java/com/griddynamics/msd365fp/manualreview/queues/service/ItemEnrichmentService.java:412]: add adds a single element.
addAll ads an array

File: backend/queues/src/main/resources/application.yml (hunk 1 above)
Comment (irinatarn) [reply backend/queues/src/main/resources/application.yml:124]: True but we will increase it with the next deployment.

File: backend/queues/src/main/resources/application-int.yml [hunk 3]
```diff
@@ -90,6 +90,7 @@ azure:
     app-service-principal-url: https://graph.microsoft.com/v1.0/servicePrincipals/${DFP_SP_ID}
     user-photo-url-template: https://graph.microsoft.com/beta/users/#user_id#/photo/$value
     timeout: PT5S
+    previous-transactions-count: 10
```
[the remark this answers is not in the document]
Comment (irinatarn) [reply backend/queues/src/main/resources/application-int.yml]: Talked offline

File: backend/queues/src/main/resources/application.yml (hunk 1 above)
Comment (satyamamit) [reply backend/queues/src/main/resources/application.yml:124]: why not same everywhere

**Review by satyamamit — APPROVED** [review]

**Pull request merged**